README.rst

```diff
@@ -124,7 +124,7 @@ Server profiles:
 Testing
 -------
 The PyKMIP test suite is composed of two parts: a unit test suite composed of
-over 600 unit tests, and an integration test suite that runs various tests
+over 750 unit tests, and an integration test suite that runs various tests
```

Comment:
nit: I would not document the number of unit tests. That will always be out of sync and difficult to maintain.